src/components/game/GameField.tsx

```diff
@@ -1,15 +1,19 @@
-import React from "react";
+import React, { useEffect } from "react";
 import { observer } from "mobx-react";
 import { useMainStore } from "../../hooks/useMainStore";
+import sound from "./win-sound.mp3";
 
 import { Card } from "../cards/Card";
 import styles from "./GameField.module.scss";
 import { BettingControls } from "../betting-chips/BettingControls";
 import { GameStage } from "../../stores/gameStore";
 import { CardSuit } from "../../types";
+import { useAudio } from "../../hooks/useAudio";
 
 export const GameField = observer(() => {
-  const { game, player, baccarat } = useMainStore();
+  const { game, player, baccarat, soundVolume, toggleSound, didWin } =
+    useMainStore();
+  const winSound = useAudio(sound, { volume: soundVolume });
```

Comment:
Instead of writing `baccarat.playerCards[2]`, etc. I'd do

```typescript
const [playerFirstCard, playerSecondCard, playerThirdCard] = baccarat.playerCards;
```